qa/smoke-test-ingest-with-all-dependencies/src/test/java/org/elasticsearch/ingest/IngestMustacheSetProcessorIT.java

```diff
@@ -1,72 +0,0 @@
-/*
```

Comment:
Because we test this already in the rest tests that are part of this module. If templating is broken then these rest tests will fail too, so it safe to remove these tests.